Makefile

```diff
@@ -16,7 +16,7 @@ unicorn/build: unicorn/CMakeLists.txt
 	mkdir -p unicorn/build
 	cd unicorn/build && cmake .. -DUNICORN_ARCH=mips -DCMAKE_BUILD_TYPE=Release
 	# Not sure why, but the second invocation is needed for fresh installs on MacOS.
-	if [ "$(uname)" == "Darwin" ]; then \
+	if [ "$(shell uname)" == "Darwin" ]; then \
```

Comment:
Ah I see! Make interprets `$` for its own variables/functions, so I should have escaped `$` and written `$$(uname)` to get the shell behaviour.

The fix you've made also works, let's merge this.